app/views/courses/ClassroomView.coffee

```diff
@@ -55,6 +54,11 @@ module.exports = class ClassroomView extends RootView
     @ownedClassrooms = new Classrooms()
     @ownedClassrooms.fetchMine({data: {project: '_id'}})
     @supermodel.trackCollection(@ownedClassrooms)
+    @levels = new Levels()
+    @levels.fetchForClassroom(classroomID, {data: {project: 'name,slug,original'}})
+    @levels.on 'add', (model) -> @_byId[model.get('original')] = model # so you can 'get' them
```

Comment:
I mainly did this for performance; a `getByOriginal` method would need to be O(1). As long as that's maintained, I'm down for being more explicit about it.